### PR TITLE
feat(efb): add adjustable brightness setting

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

* Adds an adjustable brightness setting to the flyPad

Discord username (if different from GitHub): someperson#4953

## Testing instructions

* Make sure the brightness slider works
* Make sure the brightness range is acceptable
* Make sure the brightness setting persists over MSFS restarts

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/EFB/Settings/sync.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ type SettingSync = { simVar: [name: string, type: string], propertyName: string
 function syncSetting(simVarName, simVarUnit, propertyName) {
     const propertyValue = NXDataStore.get(propertyName);
 
+    console.log(`setting ${simVarName} to ${propertyValue}`);
```
Comment (MisterChocker) [inline src/instruments/src/EFB/Settings/sync.ts:27]: console.log

Comment (Benjozork) [reply src/instruments/src/EFB/Settings/sync.ts]: Fixed

**Review by myselfaaryan — APPROVED** [review]
LGTM!

**Comment by marcman86** [comment]
Category: QA Tester
Name: marcman86#4907
Date of testing: 6.3.2021
Version of the sim: 1.13.17.0
PR Tested: #3828
Tier of testing: 1
Changes to observe: EFB brightness
Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawn C&D at EDDP 
2.) Start EFB and played with brightness slider

- at night
- at day

3.) Set it to 100% and restart MSFS
4.) Respawned and looked that it's still 100%

Issues: -

Overall Rating: Good

Conclusions of the testing: Works. Brightness levels are good. Setting persists.

Hope this helps :-)

**Comment by chrisg06** [comment]
Category: QA Trial
Name: chrisg#3898
Date of testing: 07/03/2021
Version of the sim: 1.13.17.0
PR Tested: #3828
Tier of testing: 1
Changes to observe: Adjustable brightness settings for EFB

Testing technique: Trying new feature
Steps you've made to test the Pull Request:
1. Loaded in C&D
2. tested brightness settings in different conditions
	-Day
	-Night
3. set brightness to max and restarted sim
4. checked brightness was still set to max.

Issues (If Any): N/A

Overall Rating: Good
Conclusions of the testing: Brightness is good, holds settings, works as expected. good job!

**Pull request merged**